assets/blocks/course-outline/module-block/edit.js

```diff
@@ -122,9 +141,9 @@ const EditModuleBlock = ( {
 						/>
 					</div>
 					<div className="wp-block-sensei-lms-course-outline-module__lessons-title">
-						<h3 className="wp-block-sensei-lms-course-outline__clean-heading">
+						<div className="wp-block-sensei-lms-course-outline__clean-heading">
```

Comment:
I noticed that on the frontend it's without this `div`. And since it's no more an `h3`, I think we can remove this `div`.

Reply:
Changed the `Lessons` back to `h3` per your other comment, and normalized this bit in the editor. dc1bb75